.gitignore

```diff
@@ -176,6 +176,11 @@ ruby/tests/multi_level_nesting_test_pb.rb
 ruby/tests/service_test_pb.rb
 ruby/tests/test_import_proto2_pb.rb
 ruby/tests/test_ruby_package_proto2_pb.rb
+ruby/tests/basic_test_features_pb.rb
+ruby/tests/generated_code_editions_pb.rb
+ruby/tests/repeated_field_test_pb.rb
+ruby/tests/stress_pb.rb
+ruby/tests/utf8_pb.rb
```

Comment:
`protobuf/ruby` also has `.gitignore`. So I'm wondering whether it's better to move these files under that file.
 https://github.com/protocolbuffers/protobuf/blob/main/ruby/.gitignore